faux-mgs/src/main.rs

```diff
@@ -1097,6 +1107,44 @@ async fn run_command(
                 Ok(Output::Lines(vec!["done".to_string()]))
             }
         }
+        Command::ReadComponentCaboose { component, slot, key } => {
+            let slot = match (component, slot.as_deref()) {
+                (SpComponent::SP_ITSELF, Some("active") | None) => 0,
```

Comment:
Can we make all of these arms accept the underlying integer too? So

```suggestion
                (SpComponent::SP_ITSELF, Some("active") | Some("0") | None) => 0,
```

etc?

Reply:
Sure, done in 3f568f32327a5906bfbeafac497291c63a441d36